Search/OSPlacesSearchService.swift

```diff
@@ -39,3 +39,7 @@ import Fetch
     }
 
 }
+
+private func nsErrorFromSearchError(error: SearchError) -> NSError {
+    return NSError(domain: OSSearchErrorDomain, code: error.rawValue(), userInfo: nil)
```

Comment:
`NSLocalizedDescriptionKey` might be sensible here? Not sure I'd expect canned responses to be in the API response framework.